### DISCUSSION TITLE
ESP32-C5 with 2.4 and 5 GHz dual-band Wi-Fi 6 seems to be coming...

### DISCUSSION THREAD

**Discussion by rug1024** [arendst/Tasmota#21271] [General]
On Jun 20, 2022 [ESPRESSIF announced the ESP32-C5](https://www.espressif.com/en/news/ESP32-C5) with 2.4 and 5 GHz dual-band Wi-Fi 6.
Since then nothing new had been announced. Been waiting for this chip since then...

Today I found this: https://www.electronics-lab.com/espressif-esp32-c5-test-board-features-dual-usb-c-port-and-dual-sma-connectors/
Hope it will be interesting for many of you.

**Comment by Jason2866** [comment]
The C5 is still announce ware

**Comment by joba-1** [comment]
and that is good - I hope it is a very long time still until slow iot devices unnecessarily pollute the 5GHz band :)

**Reply by Spacefish** [reply]
Shouldn't be a problem with 802.11ax (WiFi 6). As they can transmit and receive in the same frame / airtime as other devices due to RUs.

They will just get a fraction of the channel bandwidth for a single frame. So they transmit like a 500kHz wide signal and the remaining 79.5 MHz of a 80MHz wide channel can be used by another transmission.

This saves power as well, as the total TX power required is way less for such a narrow band.

**Reply by sfromis** [reply]
The possible concern with "pollution" of the 5GHz band is not so much bandwidth consumption, but number of packets, if you have over 100 such devices chatting along. As the 2.4 GHz band (already somewhat cluttered) has longer range, this tends to be a better match for having IoT devices all over the place.

**Comment by davidelang** [comment]
Even if they can decode overlapping signals, you will still have the problem of 
a strong signal from one station making it hard to hear another. Radios all have 
amplifiers in them that scale the received signal to maximize the 
signal-to-noise as you are decoding it. If you have a strong signal in one part 
of the band that gets past your input filters, you scale to that and another 
signal you are trying to decode becomes much weaker

one common term for this is desense. The fix is to have very tight RF filters, 
but that doesn't work well when you have very wide signals, let alone when you 
are talking multiple signals on the same frequency.

David Lang

**Comment by jsiddall** [comment]
I stumbled upon a Seeed Studio XIAO ESP32C5 Dual-Band module the other day so I guess they are no longer "announce ware".

I see there is an ESP32 C5 build, presumably this works on the module, but is there any support for 5 GHz yet?  I couldn't find any mentions on github.

**Reply by sfromis** [reply]
In [this Github thread](https://github.com/arendst/Tasmota/discussions/24195), Jason2866 suggests that 5 GHz should work.
And with Athom selling a [plug with ESP32-C5 preflashed with Tasmota](https://www.athom.tech/blank-1/tasmota-esp32-c5-dual-band-wi-fi-eu-plug), that also suggests that both bands works.

**Comment by arendst** [comment]
Yes, the athom works on 5G (tested myself)
